modules/test/test_calc.py

```diff
@@ -29,7 +29,7 @@ def test_c_scientific(self):
         input = Mock(group=lambda x: '2^64')
         c(self.phenny, input)
 
-        self.phenny.say.assert_called_once_with('1.84467440737096 * 10^19')
+        self.phenny.say.assert_called_once_with('1.84467441 × 10^19')
```

Comment:
I don't think it's reasonable to expect people to use Unicode here. Is there really no other way to do this?